server/app/app.js

```diff
@@ -7,7 +7,7 @@ const router = require('./routes/index');
 
 require ('dotenv').config();
 
-var whitelist = ['http://localhost:3000', 'https://learnow.netlify.com','https://learnow-be.herokuapp.com'];
```

Comment:
For some reason when i added the heroku backend link it self to whitelist, ironically it blocked it self, so im removing that